tap_linkedin_ads/client.py

```diff
@@ -114,7 +114,11 @@ def raise_for_error(response):
     message = "HTTP-error-code: {}, Error: {}".format(
                 error_code, error_description)
 
-    exc = ERROR_CODE_EXCEPTION_MAPPING.get(error_code, {}).get("raise_exception", LinkedInError)
+    if error_code not in ERROR_CODE_EXCEPTION_MAPPING and error_code > 500:
```

Comment:
Can this be `>= 500` so it doesn't rely on the exception mapping alone to catch 500 errors?

Reply:
500 and 504 are known errors as per the [linkedin-ads documentation](https://docs.microsoft.com/en-us/linkedin/shared/api-guide/concepts/error-handling?context=linkedin%2Fmarketing%2Fcontext&view=li-lms-2022-07) and that's why we defined exception class for these `status_codes` in `ERROR_CODE_EXCEPTION_MAPPING`. For errors with status codes other than 500 and 504, we directly raise `Server5xxError`. That's why here we check `> 500` because exception class for status_code 500 is available in the `ERROR_CODE_EXCEPTION_MAPPING` map.